src/main/java/com/emc/object/ObjectRequest.java

```diff
@@ -81,7 +82,19 @@ public ObjectRequest(ObjectRequest other) {
      * modify the result.
      */
     public Map<String, List<Object>> getHeaders() {
-        return new HashMap<String, List<Object>>();
+        return new HashMap<String, List<Object>>(customHeaders);
+    }
+
+    /**
+     * @param key
+     * @param value
```

Comment:
please remove empty javadoc params (they add nothing and take up space)

Reply:
✔️